Assets/Plugins/VRInteraction/Scripts/Editor/VRDefaultEditor.cs

```diff
@@ -114,4 +114,9 @@ public class VRInteractableEditor : VRDefaultEditor
     public class VRRotatableEditor : VRDefaultEditor
     {
     }
+        
+    [CustomEditor(typeof(VRTeleport))]
```

Comment:
error CS0246: The type or namespace name 'VRTeleport' could not be found (are you missing a using directive or an assembly reference?)

Reply:
sorry, files zum pushn vagessn. :)